src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -4457,11 +4457,31 @@ public static long getGCLogThreshold()
         return conf.gc_log_threshold.toMilliseconds();
     }
 
-    public static void setGCLogThreshold(int gcLogThreshold)
+    public static void setGCLogThreshold(long gcLogThreshold)
     {
         conf.gc_log_threshold = new DurationSpec.IntMillisecondsBound(gcLogThreshold);
     }
 
+    public static long getZGCLogThreshold()
+    {
+        return conf.gc_log_zgc_threshold.toMilliseconds();
```

Comment:
Not a blocker for this ticket and we do not have test coverage etc, neither I think it is a blocker but as a future reminder - I recommend adding simple unit tests to the `DatabaseDescriptorTest`.